### PR TITLE
Copy logs to workdir in archive_release()

### PR DESCRIPTION
**Pull request by david-mcmahon**

### CODE REVIEW

File: anago [hunk 1]
```diff
@@ -757,6 +763,7 @@ archive_release () {
 
   ((FLAGS_yes)) || common::askyorn -y "Archive this release on GS" || return
 
+  copy_logs_to_workdir
```
Comment (ixdy) [inline anago]: why add this here? don't we call `common::exit` right after this step, which itself will call `common::cleanexit` and then copy the logs?

Comment (david-mcmahon) [inline anago]: archive_release should get the logs as well.  Right now they aren't.  The logs should be copied into the build directory before archiving starts.

**Comment by david-mcmahon** [comment]
@enisoc, that's a good point.  I added a common::sanitize_log() to deal with the curl command-line.

**Comment by enisoc** [comment]
I'm still worried about other leaks we don't know about, or ones that sneak in later. The log is really huge and I don't think anyone can guarantee it's currently safe and will always stay safe.

Is it possible to upload the file but make the permissions only readable by the account that owns the bucket? Then it would be available to release managers, but protected from the general public.

**Comment by david-mcmahon** [comment]
@enisoc Another good suggestion.  PTAL.

**Pull request merged**